src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -217,7 +217,9 @@ class CrossDeviceLinkUI extends Component {
           <div className={style.copyLinkSection}>
             <div className={`${style.label}`}>Copy link instead:</div>
               <div className={classNames(style.actionContainer, {[style.copySuccess]: this.state.copySuccess})}>
-                <textarea className={style.linkText} ref={(textarea) => this.textArea = textarea} value={mobileUrl} />
+                <textarea className={style.linkText} ref={(element) => this.linkText = element}>
```

Comment:
did anything change?

Reply:
functionally? I don't think so. But the link text appears in the DOM this way so the tests can parse it

Reply:
saw it now. well, it break the standard I think, because it's a form element. https://stackoverflow.com/questions/5314186/javascript-get-textarea-input-via-value-or-innerhtml

Reply:
I can revert this - I found a way to pull the value out using selenium:`element.attribute('value')` rather than `element.text`

This element should probably just be a <p></p>, but I don't plan to change it now

Reply:
http://www.rubydoc.info/gems/selenium-webdriver/0.0.28/Selenium/WebDriver/Element